### PR TITLE
cleaning

### PR DESCRIPTION
**Pull request by ericgaspar**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

**Comment by l-x** [comment]
!testme

**Comment by l-x** [comment]
!testme

File: scripts/remove [hunk 1]
```diff
@@ -3,13 +3,26 @@
...
+# REMOVE SYSTEMD SERVICE 
+#=================================================
+ynh_script_progression --message="Removing system configurations related to $app..." --weight=1
+
+# Remove the service from the list of services known by YunoHost (added from `yunohost service add`)
+if ynh_exec_warn_less yunohost service status $app >/dev/null
+then
+	ynh_script_progression --message="Removing $app service integration..." --weight=1
+	yunohost service remove $app
+fi
```
Comment (Salamandar) [inline scripts/remove:18]:
```suggestion
# Remove the service from the list of services known by YunoHost (added from `yunohost service add`)
if ynh_exec_warn_less yunohost service status $app >/dev/null
then
	ynh_script_progression --message="Removing $app service integration..." --weight=1
	yunohost service remove $app
fi

ynh_script_progression --message="Removing system configurations related to $app..." --weight=1
```

**Comment by l-x** [comment]
!testme

File: scripts/remove (hunk 1 above)
Comment (l-x) [reply scripts/remove:18]: Thanks for you contribution. If I'm not wrong you just moved the line `ynh_script_progression --message="Removing system configurations related to $app..." --weight=1` after the `if` block!?

Comment (Salamandar) [reply scripts/remove:18]: Yeah, indeed :) It’s just to have more consistent progression messages: the "removing app service integration" message will be visible while the other system configurations will be actually removed. It’s a nitpick :D

**Comment by l-x** [comment]
!testme

**Comment by Salamandar** [comment]
LGTM, let’s merge !

**Pull request merged**